File: python/1002 - 1030/1013 - O maior.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)
maior_AB = (a+b+abs(a-b))/2
maior_C = (maior_AB+c+abs(maior_AB - c))/2
print(f'{maior_C:.0f} eh o maior')

File: Pascal/Beecrowd/1013 - O maior.pas
Var
  a, b, c, maiorab, maior: real;
Begin
	readln(a);
	readln(b);
	readln(c);
	maiorab:= ((a + b + abs(a - b)) / 2);
	maior:= ((maiorab + c + abs(maiorab - c)) / 2);
	writeln(maior:0:0,' eh o maior');   
End.